apps/i18n/javalab/en_us.json

```diff
@@ -21,9 +21,11 @@
   "internalException": "Oops. We hit an error on our side. Try running your program again. If this error continues, contact us at support@code.org. Be sure to include the connection ID {connectionId} in your message.",
   "internalRuntimeException": "Oops. We hit an error on our side while running your program. Try running it again. If this error continues, contact us at support@code.org. Be sure to include the connection ID {connectionId} in your message.",
   "javaExtensionMissing": "Invalid file name. File names must end in '.java'.",
+  "markResolved": "Mark Resolved",
```

Comment:
this may be outside your control, but can we change this to just 'resolve' and 'unmark resolved' to 're-open'? Google docs says 'mark as resolved' and 're-open' for what it's worth. @moneppo for thoughts

Reply:
I changed it :) Can change back in future if needed.